File: mo-member/mo-member-service/src/main/java/ink/ckx/mo/member/mapper/MemberUserMapper.java
package ink.ckx.mo.member.mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import ink.ckx.mo.member.api.model.entity.MemberUser;
import org.apache.ibatis.annotations.Mapper;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/03
 */
@Mapper
public interface MemberUserMapper extends BaseMapper<MemberUser> {
}

File: mo-member/mo-member-service/src/main/kotlin/ink/ckx/mo/member/mapper/MemberUserMapper.kt
package ink.ckx.mo.member.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import ink.ckx.mo.member.api.model.entity.MemberUser
import org.apache.ibatis.annotations.Mapper

@Mapper
interface MemberUserMapper : BaseMapper<MemberUser>